server/game/cards/locations/07/blackbetha.js

```diff
@@ -1,53 +1,35 @@
 const DrawCard = require('../../../drawcard.js');
 
 class BlackBetha extends DrawCard {
-    //Just used Iron Fleet Scout as template.  Overly verbose, but easy to understand.  Could be refactored into something more concise (a la The Valyrian).
-
     setupCardAbilities(ability) {
         this.persistentEffect({
             match: card => card.name === 'Ser Davos Seaworth',
             effect: ability.effects.addKeyword('renown')
         });
         this.action({
             title: 'Kneel to give attacking character +X STR',
```

Comment:
Now you just need a cost (so that it actually kneels), and a condition (so that it can only initiate if it actually has the ability to change the game state. Something like this should work:

```javascript
            title: 'Kneel to give attacking character +X STR',
            condition: () => this.game.currentChallenge && this.calculateStrength() >= 1,
            cost: ability.costs.kneelSelf(),
            target: {
```